libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -188,18 +188,19 @@ export function trigger({ element, event, type } = {}) {
 
 export const yieldToMain = () => new Promise((resolve) => { setTimeout(resolve, 0); });
 
-export const lanaLog = ({ message, e = '' }) => {
+export const lanaLog = ({ message, e = '', tags = 'errorType=default' }) => {
   const url = getMetadata('gnav-source');
   window.lana.log(`${message} | gnav-source: ${url} | href: ${window.location.href} | ${e.reason || e.error || e.message || e}`, {
     clientId: 'feds-milo',
     sampleRate: 1,
+    tags,
   });
 };
 
-export const logErrorFor = async (fn, message) => {
+export const logErrorFor = async (fn, message, tags) => {
   try {
     await fn();
   } catch (e) {
-    lanaLog({ message, e });
+    lanaLog({ message, e, tags });
```

Comment:
Nit: we could add a unit test for this in `utilities.test.js`, just calling the method with an erroneous function, to satisfy codecov.

Reply:
Done, but codecov is still not happy